content/en/docs/migration/flux-v1-migration.md

```diff
@@ -19,6 +19,12 @@ design.
 This may at times mean that you have to make adjustments to the way your
 current cluster configuration is structured. If you are in this situation
 and need help, please refer to the [support page](/support/).
+
+One of the interesting features of flux2 is the "server-side validations" of the manifests. 
+It is important to note that the manifests are applied to the cluster only if the validation passes.
+If one manifest has multiple resources, an error in one of the resources would also prevent other resources from getting applied. 
+This was not the behavior with flux1
```

Comment:
```suggestion
One key change in Flux v2 is "server-side apply" that enforces strict
validation of the manifests.  It is important to note that manifests are
applied atomically to the cluster only if YAML validation passes. If one
Kustomization has multiple resources, an error in any one of the resources
will also prevent other resources in that group from getting applied. This is
a breaking change from Flux v1.
```

I think this conveys the advantage of doing things this way without papering over, the words "breaking change" probably belong in whatever version of this we accept 👍 

Thanks for suggesting this update